izumi-reflect/izumi-reflect/src/main/scala/izumi/reflect/macrortti/LTTRenderables.scala

```diff
@@ -227,7 +230,62 @@ object LTTRenderables {
     override def r_SymName(sym: SymName, hasPrefix: Boolean): String = {
       Long.r_SymName(sym, hasPrefix)
     }
+
     override def prefixSplitter: String = "."
   }
 
-}
+  object ScalaStyledLambdas extends Long {
+
+    override def prefixSplitter: String = "."
+
+    override implicit lazy val r_LambdaParameterName: Renderable[SymName.LambdaParamName] = new Renderable[SymName.LambdaParamName] {
+      override def render(value: SymName.LambdaParamName): String = "_"
+    }
+
+    override implicit lazy val r_Lambda: Renderable[Lambda] = new Renderable[Lambda] {
+      override def render(value: Lambda): String = {
+        s"${value.output.render()}"
+      }
+    }
+
+    override implicit lazy val r_Variance: Renderable[Variance] = new Renderable[Variance] {
+      override def render(value: Variance): String = value match {
+        case Variance.Invariant => ""
+        case Variance.Contravariant => "-"
+        case Variance.Covariant => "+"
+      }
+    }
+
+    override implicit lazy val r_NameRefRenderer: Renderable[NameReference] = new Renderable[NameReference] {
+      override def render(value: NameReference): String = {
+        val r = r_SymName(value.ref, value.prefix.isDefined)
+
+        val rr = value.boundaries match {
+          case _: Boundaries.Defined =>
+            s"$r ${value.boundaries.render()}"
+          case Boundaries.Empty =>
+            r
+        }
+
+        value.prefix match {
+          case Some(p) =>
+            s"${p.render()}$prefixSplitter$rr"
+          case None =>
+            rr
+        }
+      }
+    }
+
+    override implicit lazy val r_Boundaries: Renderable[Boundaries] = new Renderable[Boundaries] {
+      override def render(value: Boundaries): String = value match {
+        case Boundaries.Defined(bottom, top) =>
+          (bottom.render(), top.render()) match {
+            case (bottomRendered, topRendered) if bottomRendered == "scala.Nothing" => s"<: $topRendered"
```

Comment:
This is kinda dirty. We already have all the necessary constants in `LightTypeTagInheritance`, please reuse them.

Reply:
Yep, removed my hardcode